.github/workflows/pr-stale.yml

```diff
@@ -0,0 +1,22 @@
+name: "Stale issues handler"
+
+on:
+  workflow_dispatch: # allows workflow to be manually triggered from GitHub UI
+  schedule:
+    - cron: "0 5 * * *" # runs at 5am UTC
+
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/stale@v8
+        with:
+          repo-token: ${{ secrets.GITHUB_TOKEN }}
+          debug-only: true
+          days-before-issue-stale: 1
+          days-before-issue-close: 1
+          stale-issue-label: "no-issue-activity"
+          labels-to-remove-when-unstale: "no-issue-activity"
+          stale-issue-message: |
+            This issue is stale because it has been open 1 day with no activity.
+            Please comment or this issue will be automatically closed in 1 day.
```

Comment:
Suggesting more detailed messages so users have more context about why an issues is marked as stale os is being auto closed:

```suggestion
          stale-issue-message: |
            👋 Hi there!
            This issue has been automatically marked as stale because there wasn't any activity in the past 
            3 weeks.
            * Please check if it is still relevant in latest version of Devopness
            * If so, please update the issue with new details
            If this issue remains without activity for another week (completing a month without 
            activity) be automatically closed.
          close-issue-message: |
            ❌ This issue was closed because it has been open and inactive for a month.
            If you feel this is still a high-priority issue, or are interested in contributing, please 
            open a new issue or pull request linking to this one, for context.
```

Reply:
I've improved the stale issue message and added the closed issue message as suggested.